### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Hi there 👋
+
+The organization aims to explain modern and complex ML tech to the community. Understanding any ML component comprehensively involves of three things:
```
Comment (AakashKumarNain) [inline profile/README.md]: Org is a powerful word. I will avoid using it as much as possible. Maybe something like this?

We aim to explain new and complex ML tech to the community. This is our way of giving back to the awesome ML community. Understanding....

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+* Underlying math
+* Conceptual framework of the component
+* Corresponding code 
+
+We're starting with a series on Diffusion models where there will be deep dives into the fundamental math behind them as well as self-contained and readable code.
```
Comment (AakashKumarNain) [inline profile/README.md]: where we will deep dive*

File: profile/README.md (hunk 1 above)
Comment (sayakpaul) [inline profile/README.md]:
```suggestion
We aim to explain new and complex ML tech to the community. This is our way of giving back to the community. Understanding any ML component comprehensively involves three things:
```

File: profile/README.md (hunk 2 above)
Comment (sayakpaul) [inline profile/README.md]:
```suggestion
We're starting with a series on Diffusion models where we will dive deep into the fundamental math behind them as well as provide self-contained and readable code. 
```

**Comment by sayakpaul** [comment]
@AakashKumarNain see if the suggestions look good.

File: profile/README.md (hunk 1 above)
Comment (AakashKumarNain) [reply profile/README.md]: LGTM!

File: profile/README.md (hunk 2 above)
Comment (AakashKumarNain) [reply profile/README.md]: LGTM!

**Pull request merged**